tests/foreman/ui_airgun/test_contentview.py

```diff
@@ -2764,3 +2767,297 @@ def test_positive_search_module_streams_in_content_view(session, module_org):
+        role=role,
+        search=None
+    ).create()
+    # create environment permissions with read only and promote access
+    # to content views
+    env_permissions_entities = entities.Permission(
+        resource_type='Katello::KTEnvironment').search()
+    user_env_permissions = [
+        'promote_or_remove_content_views_to_environments',
+        'view_lifecycle_environments'
+    ]
+    user_env_permissions_entities = [
+        entity
+        for entity in env_permissions_entities
+        if entity.name in user_env_permissions
+    ]
+    entities.Filter(
+        organization=[module_org],
+        permission=user_env_permissions_entities,
+        role=role,
+        # allow access only to the mentioned here environments
+        search='name = {0} or name = {1}'.format(ENVIRONMENT, lce.name)
+    ).create()
+    # create a user and assign the above created role
+    entities.User(
+        default_organization=module_org,
+        organization=[module_org],
+        role=[role],
+        login=user_login,
+        password=user_password,
+        mail='test@test.com',
+    ).create()
+    create_sync_custom_repo(module_org.id, repo_name=repo_name)
+    # create a content view with the main admin account
+    with session:
+        session.contentview.create({'name': cv_name})
+        assert session.contentview.search(cv_name)[0]['Name'] == cv_name
+        session.contentview.add_yum_repo(cv_name, repo_name)
+        # Copy the CV
+        session.contentview.copy(cv_name, cv_copy_name)
+        assert session.contentview.search(cv_copy_name)[0]['Name'] == cv_copy_name
+    # login as the user created above
+    with Session(test_name, user=user_login, password=user_password) as session:
+        with raises(NavigationTriesExceeded):
+            session.organization.create({
+                'name': gen_string('alpha'),
+                'label': gen_string('alpha'),
+            })
+        # assert the user can view all the content views created
+        # by admin user
+        if bz_bug_is_open(1652938):
+            try:
+                session.contentview.search('')
+            except (NavigationTriesExceeded, NoSuchElementException):
+                session.browser.refresh()
+        assert session.contentview.search(cv_name)[0]['Name'] == cv_name
+        assert session.contentview.search(cv_copy_name)[0]['Name'] == cv_copy_name
+        # assert that the user can delete a content view
+        session.contentview.delete(cv_copy_name)
+        assert not session.contentview.search(cv_copy_name)
+        # check that cv tabs are accessible
+        cv = session.contentview.read(cv_name)
+        for tab_name in [
+            'details',
+            'versions',
+            'repositories',
+            'filters',
+            'puppet_modules',
+            'docker_repositories',
+            'ostree_content'
+        ]:
+            assert cv[tab_name] is not None
```

Comment:
Use `cv.get(tab_name)` instead, as in negative case it won't be `None`, but `KeyError` will be risen.